.travis.yml

```diff
@@ -8,4 +8,4 @@ before_install:
 script:
   - docker ps | grep -q radicale
   - sleep 10
-  - curl -sSL http://127.0.0.1:5232 | grep -q "Radicale works"
+  - curl -sSL http://127.0.0.1:5232 | grep -q "Login"
```

Comment:
Same with this command, again not sure how travis is using it, but I use something like `curl --fail http://localhost:5232 || exit 1` for doing status checks

Reply:
Nice trick! commit pushed.